load-data-from-kafka/persist/model.bal

```diff
@@ -0,0 +1,23 @@
+import ballerina/persist as _;
+import ballerina/time;
+
+public type LeadAnalyticsData record {|
+    readonly string id;
+    string LeadID;
```

Comment:
shall we use simple letters at the start

Reply:
updated.